typescript/helloworld/package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "@abacus-network/helloworld",
   "description": "A basic skeleton of an Abacus app",
-  "version": "0.4.0",
+  "version": "0.5.0",
```

Comment:
This was a mistake because helloworld was prematurely bumped to 0.4.0. 
Already published as 0.5.0 unfortunately